src/test/java/org/dcsa/bkg/model/transferobjects/DocumentPartyTOTest.java

```diff
@@ -19,20 +20,22 @@
 class DocumentPartyTOTest {
```

Comment:
missing test case for displayed address larger then 250 characters

Reply:
It's already there, line 55
`
  @Test
  @DisplayName(
      "DocumentPartyTO should throw error if displayedAddress length exceeds max size of 250.")
`